api-notes.md

```diff
@@ -0,0 +1,325 @@
+  <Breadcrumb as={StyledLink} href="/components">Parent</Breadcrumb>
+  <Breadcrumb current>Current Page</Breadcrumb>
+</Breadcrumbs>
+
+// Lower level API
+<Breadcrumbs>
+  <BreadcrumbList>
+    <BreadcrumbItem>
+      <a href="/">Home</a>
+      <FancyIcon aria-hidden />
+    </BreadcrumbItem>
+    <BreadcrumbItem>
+      <StyledLink href="/components">Parent</StyledLink>
+      <FancyIcon aria-hidden />
+    </BreadcrumbItem>
+    <BreadcrumbItem>
+      <span>Current Page</span>
+    </BreadcrumbItem>
+  </BreadcrumbList>
+</Breadcrumbs>
+```
+
+## ❓: Button
+
+TBD
+
+## 🛠: Carousel
+
+```jsx
+<Carousel>
+  <CarouselSlides>
+    <CarouselSlide>Slide 1</CarouselSlide>
+    <CarouselSlide>Slide 2</CarouselSlide>
+    <CarouselSlide>Slide 3</CarouselSlide>
+  </CarouselSlides>
+  <CarouselArrows />
+  <CarouselIndicators />
+</Carousel>
+```
+
+```ts
+type CarouselProps = {
+  activeIndex: // starting index for controlled components
+  // if multiple slides are visible, this positions the currently active slide.
+  // default: center
+  activeSlidePosition: 'center' | 'start' | 'end';
+  autoPlay: boolean | number; // number = speed in MS
+  controls: React.RefObject; // turns the slider into a control for another slider.
+  defaultActiveIndex: number; // default starting index for uncontrolled components
+  disabled: boolean;
+  draggable: boolean; // default: true
+  // whether or not the slider should be an infinite loop
+  // imagining we'd need our component to control this if the user wants to
+  // control animation between the last and first slides
+  // (might also affect how indicators are rendered + arrow behavior)
+  loopSlides: boolean;
+  onChange(event, trigger): void;
+  orientation: 'horizontal' | 'vertical'; // default: 'horizontal'
+  rtl: boolean; // sets read/swipe mode to RTL
+  slidesShown?: number; // default: 1
+};
+```
+
+## 🛠: Disclosure
+
+> NOTE: This should follow the Accordion, as it's essentially the same basic behavior with less flexibility and no custom focus management.
+
+```jsx
+<Disclosure>
+  <DisclosureItem>
+    <DisclosureTrigger>Open the box</DisclosureTrigger>
+    <DisclosureContent>Some content</DisclosureContent>
+  </DisclosureItem>
+  <DisclosureItem>
+    <DisclosureTrigger>Open the box</DisclosureTrigger>
+    <DisclosureContent>Some content</DisclosureContent>
+  </DisclosureItem>
+</Disclosure>
+```
+
+## ❓: Feed
+
+```jsx
+// suspense component? Maybe? Does this even work?
+function Example() {
+  const posts = resource.posts.read();
+  return (
+    <Feed fallback={<Loading />} loadingNext={<LoadingNext />}>
+      {posts.map(post => (
+        <FeedItem key={post.id}>
+          <h1>{post.title}</h1>
+          <a href={post.src}>
+            Read More<VisuallyHidden> about {post.title}</VisuallyHidden>
+          </a>
+        </FeedItem>
+      ))}
+    </Feed>
+  );
+}
+```
+
+## ❓: Grid
+
+TBD
+
+## ❓: Link
+
+TBD
+
+## 🛠: Listbox
+
+```jsx
+<Listbox>
```

Comment:
Not sure what this wrapper `Listbox` component is for? Much like the disclosure one. Listboxes aren't related to eachother any more than `<select>`.